code/render-code.sh

```diff
@@ -1,12 +1,12 @@
-#/bin/sh
+#!/bin/sh
 NAME=${1-movies}
 TARGETPATH=${2-/tmp}
 TARGET="$TARGETPATH/$NAME"
 if [ ! -d $TARGET ]; then
-    git clone https://github.com/neo4j-graph-examples/$name $TARGET
+    git clone https://github.com/neo4j-graph-examples/$NAME $TARGET
 fi
 
-QUERY=`grep -e '^\(:query:\| .*\+$\)' $TARGET/README.adoc | cut -d' ' -f2- | sed -e 's/\+$//g'` 
+QUERY=`pcregrep -M '^:query:|.*\+\s*$' $TARGET/README.adoc | cut -d' ' -f2- | sed -e 's/\+$//g'` 
```

Comment:
does pcregrep need to be installed first?

Reply:
if so, that's a bummer :( I assumed not